packages/camera/camera/example/lib/main.dart

```diff
@@ -313,6 +313,11 @@ class _CameraExampleHomeState extends State<CameraExampleHome>
   }
 
   Widget _exposureModeControlRowWidget() {
+    final ButtonStyle style = TextButton.styleFrom(
```

Comment:
The first and second button change based on `ExposureMode.auto` and `ExposureMode.locked`. I think you might have missed the styles were slightly different.

Reply:
Good catch! I'll fix it.

Reply:
Done. PTAL